topics/server-integrate-database.topic

```diff
@@ -867,21 +868,35 @@
             </procedure>
         </chapter>
         <p>
-            With this, you have successfully completed integrating a database into your application.
-        </p>
-        <p>
-            The <code>FakeTaskRepository</code> type is no longer needed in production code, but is still valuable for
-            testing. Therefore, you can move it to the testing source set, in
-            <path>src/test/com/example</path>
-            .
+            With this, you have successfully completed integrating a database into your application. However, if you
+            run the tests, you will find that they are now failing.
         </p>
-        <p>
-            The final project structure should look something like this:
-        </p>
-        <img src="tutorial_server_db_integration_src_folder.png"
-             alt="The src folder shown within the Project View in intelliJ IDEA"
-             border-effect="line"
-             width="400"/>
+        <procedure title="Configure the test module">
+            <step>
+                For each test in the
+                <path>ApplicationTest.kt</path>
+                file, use the <code>MapApplicationConfig()</code> function to override the default application
+                configuration and use a custom module:
+                <code-block
+                        src="snippets/tutorial-server-db-integration/src/test/kotlin/com/example/ApplicationTest.kt"
+                        lang="kotlin"/>
+            </step>
+            <step>
+                <p>
+                    As the <code>FakeTaskRepository</code> type is no longer needed in production code, you can move
```

Comment:
Why was it even there if it's a fake implementation?

Reply:
It's considered useful for testing the routes only, though I do think having `Fake` in the name can be misleading. Perhaps we can call it `TestTaskRepository` or `InMemoryTaskRepository`?